project2/style.css

```
@@ -0,0 +1,66 @@
+.title {
+    font-family: 'ZCOOL QingKe HuangYou', cursive;
+    color: white;
+    font-size: 19pt;
```

For browsers, you want to provide your font-size value in pixels:
```suggestion
    font-size: 19px;
```
There are some alternatives, but for these projects, I'm asking y'all to use pixels exclusively. Note that 19px is not the same as 19pt, so you'd probably want to increase this value, too (25px is probably close to what size you have now).